tests/robottelo/decorators/test_host.py

```diff
@@ -12,139 +14,197 @@
 
 
 class GetHostOsVersionTestCase(TestCase):
+    """Tests for _get_host_os_version version"""
+
     def setUp(self):
+        """Mocking ssh"""
         self._patcher = mock.patch(
             'robottelo.decorators.host.ssh.command')
         self._command = self._patcher.start()
 
     def tearDown(self):
+        """Stop mock created on setUp method"""
         self._patcher.stop()
 
     def assert_rhel_version(self, ssh_version, parsed_version):
+        """
+        Encapsulate assertion logic regarding host os parsing
+        :param ssh_version: version returned from ssh
+        :param parsed_version: parsed version
+        """
         self._command.return_value.stdout = [ssh_version]
-        self.assertEqual(parsed_version, host._get_host_os_version())
+        self.assertEqual(
+            parsed_version,
+            host._get_host_os_version.__wrapped__()
+        )
         self._command.assert_called_once_with('cat /etc/redhat-release')
 
-    def test_rhel_6(self):
+    def test_rhel_major_version_parsing(self):
+        """Version semantic example: 1.2.3
+        1 is major
```

Comment:
The documentation generated for this docstring will be something like `Version semantic example: 1.2.3 1 is major 2 is minor 3 is patch`

I think this docstring should be something like: `Check if can parse major version.`